core/src/com/unciv/ui/worldscreen/AlertPopup.kt

```diff
@@ -184,11 +184,27 @@ class AlertPopup(val worldScreen: WorldScreen, val popupAlert: PopupAlert): Popu
                 val wonder = worldScreen.gameInfo.ruleSet.buildings[popupAlert.value]!!
                 addGoodSizedLabel(wonder.name)
                 addSeparator()
+                if(ImageGetter.wonderImageExists(wonder.name)) {    // Wonder Graphic exists
+                    if(worldScreen.stage.height * 3 > worldScreen.stage.width * 4) {    // Portrait
```

Comment:
For next time: Consider whether `CameraStageBaseScreen.isPortrait()` might be sufficient.